packages/nextjs-mf/src/plugins/container/custom-jsonp.js

```diff
@@ -0,0 +1,195 @@
+
+          const version = lib[versionKey];
+          console.log(version)
+          if (!version.loaded) {
+           version.loaded = 1;
+            console.log('loading', libKey, versionKey, r);
+            await version
+              .get()
+              .then((f) => {
+                console.log('loaded', libKey, versionKey, r);
+                version.get = () => f;
+                version.loaded = 1;
+              });
+          }
+        }
+      }
+    })
+    .then((r) => {
+      console.log('webpack is done negotiating dependency trees', cnn);
+      console.log('number of entry points to inveert startup',chunkQueue.length);
+      console.log('startup inversion in progress', chunkQueue);
+while (chunkQueue.length > 0) {
+      chunkLoadingGlobal.push(chunkQueue.shift());
+
+  //var item = chunkQueue.shift(); // Remove and get the first item from the queue
+  // console.log('calling deferred entry queue item:', item);
+  // webpackJsonpCallback.apply(null, [null].concat([item]));
+ // console.log('deferred push call:', item[0]);
+  // webpackJsonpCallback.apply(chunkLoadingGlobal, item);
+ //    webpackJsonpCallback.call(chunkLoadingGlobal,0, item);
+
+      //      webpackJsonpCallback.apply(
+      //   null,
+      //   [null].concat(Array.prototype.slice.call(item))
+      // );
+      //
+      // return webpackJsonpCallback.apply(chunkLoadingGlobal, item);
+}
+    });
+}
+
+chunkLoadingGlobal.forEach(function (item, index) {
+  asyncOperation().then(() => {
+  // console.log('forEach:', index, item);
+    webpackJsonpCallback.call(null, 0, item);
+    chunkLoadingGlobal.push.bind(chunkLoadingGlobal)(item);
+  });
+});
+
+
+chunkLoadingGlobal.push = (function (originalPush) {
+  return function () {
+
+  console.log('push catcher called push:', arguments[0][0]);
+  console.log('deffered queue', chunkQueue);
+  console.log('deffered report', resport);
+  var matchesOrStartsWith = arguments[0][0].some(function (item) {
+    return resport.some(function (resportItem) {
+    console.log('matcher', item, resportItem);
+      return item === resportItem || item.indexOf(resportItem) === 0;
+    });
+  });
+
+console.log('matchesOrStartsWith', matchesOrStartsWith);
+
+console.log('queue push hook: currently processing', arguments[0][0]);
+
+    if(!matchesOrStartsWith && arguments[0][0].includes('main')){
+    console.log('thinks its main');
+    resport = Array.prototype.concat.apply(resport, arguments[0][0]);
+    return Array.prototype.push.apply(chunkQueue, arguments);
+
+    return chunkQueue.push(arguments);
+    //   return chunkQueue.push(()=>{
+    // if(cnn === 'home_app'){
+    //
+    //   console.log('push queue main', arguments[0][0]);
+    //   }
+    //    return webpackJsonpCallback.apply(
+    //       null,
+    //       [null].concat(Array.prototype.slice.call(arguments))
+    //     );
+    //   })
+    }
+     if(!matchesOrStartsWith && arguments[0][0].some(item => item.startsWith('pages/'))){
```

Comment:
lookups need to get entrypoint chunks. 

avoid looping in runtime. webpack can print out startup stuff